### DISCUSSION TITLE
Suggested Table replacement?

### DISCUSSION THREAD

**Discussion by andioneto** [twilio-labs/paste#782] [Q&A] [answered]
Hi folks 👋 @vnguyen94 and I are working on some shared components between Numbers Console and VNV Console, which will live in the `numbers-toolbox` repository.
Since the components in the front end package need to be usable outside of Twilio Console (e.g. in Monkey), we cannot use the legacy table components as a patch. 
Could you please point us towards another design system we can use temporarily that might be a good fit?
I spent some time reviewing [Vercel](https://vercel.com/design/table) and it seems like it might be a good match.

Thank you again in advance!

**Comment by corinne-nagel** [comment]
Hi @andipants12 I'm checking with the team about when the table components will be ready to go.  When do you need it by?

Will let you know as soon as I hear back!

**Comment by corinne-nagel** [comment, marked as answer]
I got an update for you @andipants12 ! We'll have an alpha component with super light documentation in the next sprint, and we'll be doing reviews with the committee at the same time. If you want, you can start using it to test it out and give us feedback 🙏

Oh, and if you have anything you/your team particularly likes about the Vercel Table, please let us know!

**Reply by andioneto** [reply]
Ok great, thank you so much @corinne-nagel! I'll check in with folks and get back to you.
